lib/ansible/modules/windows/win_robocopy.ps1

```diff
@@ -92,74 +93,61 @@ $result.flags = $flags
 
 $robocopy_output = ""
 $rc = 0
-If ($check_mode -eq $true) {
-    $robocopy_output = "Would have copied the contents of $src to $dest"
-    $rc = 0
-}
-Else {
-    Try {
-        &robocopy $robocopy_opts | Tee-Object -Variable robocopy_output | Out-Null
-        $rc = $LASTEXITCODE
-    }
-    Catch {
-        $ErrorMessage = $_.Exception.Message
-        Fail-Json $result "Error synchronizing $src to $dest! Msg: $ErrorMessage"
-    }
+Try {
+
+    &robocopy $robocopy_opts | Tee-Object -Variable robocopy_output | Out-Null
```

Comment:
Just for the record - I'm traditionally a Linux admin, and this is some of the only powershell I've written. So if something's weird, might be best to assume I just didn't know what I was doing.